src/components/FilterPane/FilterPane.jsx

```diff
@@ -187,189 +193,213 @@ const FilterPane = ({ mermaidUserData }) => {
                 >
                   <input
-                    id={method.name}
                     type="checkbox"
-                    name={method.name}
-                    onChange={handleMethodFilter}
-                    checked={methodFilters.includes(method.name)}
-                    onClick={(e) => e.stopPropagation()}
+                    name="yourData"
+                    id="yourData"
+                    onChange={handleYourDataFilter}
+                    checked={showYourData}
                   />
-                  <StyledLabel htmlFor={method.name} onClick={(e) => e.stopPropagation()}>
-                    {method.description}
+                  <StyledLabel htmlFor="yourData" onClick={(e) => e.stopPropagation()}>
+                    {filterPane.yourData}
                   </StyledLabel>
                 </StyledClickableArea>
-              </li>
-            ))}
-          </StyledUnorderedList>
+              </StyledCategoryContainer>
+            </>
+          ) : null}
+          <StyledHeader>Data sharing</StyledHeader>
+          <StyledCategoryContainer>
+            <StyledClickableArea
+              onClick={() => handleDataSharingFilter({ target: { checked: !dataSharingFilter } })}
+            >
+              <input
+                type="checkbox"
+                name="dataSharing"
+                id="dataSharing"
+                onChange={handleDataSharingFilter}
+                checked={dataSharingFilter}
+                onClick={(e) => e.stopPropagation()}
+              />
+              <StyledLabel htmlFor="dataSharing" onClick={(e) => e.stopPropagation()}>
+                {filterPane.dataSharing}
+              </StyledLabel>
+            </StyledClickableArea>
+          </StyledCategoryContainer>
+          <StyledHeader>Methods</StyledHeader>
+          <StyledMethodListContainer>
+            <StyledUnorderedList>
+              {COLLECTION_METHODS.map((method) => (
+                <li key={method.name}>
+                  <StyledClickableArea
+                    onClick={() =>
+                      handleMethodFilter({
+                        target: {
+                          name: method.name,
+                          checked: !methodFilters.includes(method.name),
+                        },
+                      })
+                    }
+                  >
+                    <input
+                      id={method.name}
+                      type="checkbox"
+                      name={method.name}
+                      onChange={handleMethodFilter}
+                      checked={methodFilters.includes(method.name)}
+                      onClick={(e) => e.stopPropagation()}
+                    />
+                    <StyledLabel htmlFor={method.name} onClick={(e) => e.stopPropagation()}>
+                      {method.description}
+                    </StyledLabel>
+                  </StyledClickableArea>
+                </li>
+              ))}
```

Comment:
What do you think to breaking out things like this into separate components or functions. You could create something like a `renderCollection()` function or define `.map`'s callback function outside of the large block of JSX.

Reply:
Yeah I think it could make the JSX block a lot cleaner. I can clean this up a bit